tests/test_pandas_extension.py

```diff
@@ -29,9 +33,19 @@
         "data_for_grouping",
         "data_for_sorting",
         "data_missing_for_sorting",
+        "data_repeated",
     ],
 )
 
+if sys.version_info >= (3, 6):
```

Comment:
Six?

Reply:
No, six only distinguishes between Python 2 and 3. This is a 3.6+ feature that is not available in 3.5.

Reply:
You're right. I thought I saw six variables to distinguish between py3 versions. I was wrong, though.